### DISCUSSION TITLE
Register indexer after unregister

### DISCUSSION THREAD

**Discussion by WayneWayner** [subquery/subql#1045] [Frontier TestNet - Bug Reporting]
If shortly after unregistering your indexer you want to register it again, this will not work. This is ok since the action has first to settle. However, it would be nice if the error is also reported in the 8000 dashboard. Currently nothing happens, but the error is displayed in the console.
![sq_unregister_register](https://user-images.githubusercontent.com/60453944/169424877-15f0428e-f507-4fd5-bb6a-d8c2816f0f76.png)

**Comment by WayneWayner** [comment]
(Bonus) Design Bug:

Seems like there is also a small cosmetic issue when you shrink the window size. The Commission Rate field is out of the box.

**Comment by mzxyz** [comment]
Thanks for reporting this issue, the unregistered indexer can not register immediately. We do need to check the connected account and display some useful information, thanks.
